x-pack/plugins/viz_editor/editorConfigRegistry.ts

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Just FYI, I thought our style-guide said that file names should be snake_cased, but I can't seem to find that. Anyway, Kibana file names are generally snake_cased. I almost always have dash-cased my file names, so this has been challenging to remember.

Reply:
You are right, I also recall Inread that somewhere. Will change it 👍